compiler/x/codegen/OMRLinkage.hpp

```diff
@@ -378,7 +379,11 @@ class OMR_EXTENSIBLE Linkage : public OMR::Linkage
          }
       }
 
-   static TR_X86OpCodes movOpcodes[NumMovOperandTypes][NumMovDataTypes];
+   static inline TR_X86OpCodes movOpcodes(TR_MovOperandTypes oprandType, TR_MovDataTypes dataType)
+      {
+      TR_ASSERT(TR::Compiler->target.is64Bit() || dataType != TR_MovDataTypes::Int8, "MOV Int64 should not occur on X86-32");
```

Comment:
I find the use of Int4/Int8/etc consistent with the instruction opcodes and somewhat ambiguous, and in your assert message I see you fell into the same trap!  Please change `Int64` to `Int8`.